Assignment4/src/controllers/UserControoler.js

```diff
@@ -0,0 +1,13 @@
+const { secretKey, mockData } = require("../utils/helperFunction");
+const jwt = require("jsonwebtoken");
+
+const userController = (req, res) => {
+  const check = jwt.verify(req.token, secretKey);
```

Comment:
"check" is not an apt name for this particular literal. Use sensible name as isTrue, isValid etc.

Reply:
Variable name changed as per convention